tests/HeaderDataTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
```

Comment:
It would be good to add a file header describing what's being tested in this file.